lib/utils/matrix_sdk_extensions/filtered_timeline_extension.dart

```diff
@@ -27,19 +27,75 @@ extension IsStateExtension on Event {
           type != EventTypes.RoomMember ||
           room.joinRules != JoinRules.public ||
           content.tryGet<String>('membership') == 'ban' ||
-          stateKey != senderId);
+          stateKey != senderId) &&
+      !isSomeoneChangeDisplayName() &&
+      !isSomeoneChangeAvatar() &&
+      !isGroupNameChangeWhenCreate() &&
+      !isGroupAvatarChangeWhenCreate() &&
+      !isJoinedByYourself() &&
+      !isActivateEndToEndEncryption() &&
+      !isInviteWhenCreate();
 
   static const Set<String> importantStateEvents = {
     EventTypes.Encryption,
     EventTypes.RoomCreate,
     EventTypes.RoomMember,
     EventTypes.RoomTombstone,
     EventTypes.CallInvite,
+    EventTypes.RoomName,
+    EventTypes.RoomAvatar,
   };
 
   bool get isState => !{
         EventTypes.Message,
         EventTypes.Sticker,
         EventTypes.Encrypted
       }.contains(type);
+
+  bool isSomeoneChangeDisplayName() {
+    return stateKey != null &&
```

Comment:
check if someone changes their displayName, the `stateKey` is the userId of user have changed avatar, and then compare the old displayname with new displayname